ocs_ci/templates/ocs-deployment/provider-mode/storage_consumer.yaml

```diff
@@ -1,15 +1,18 @@
 apiVersion: ocs.openshift.io/v1alpha1
 kind: StorageConsumer
 metadata:
+  namespace: openshift-storage
```

Comment:
Why are changes to this template necessary?

Reply:
this CR was created 7 of April. I missed the namespace initially, assuming it will be changed frequently and hence be set in python script. This default namespace removes couple lines from code